Sources/SwiftProtobuf/ProtoNameResolvers.swift

```diff
@@ -12,22 +12,20 @@
 /// This type contains helper functions to resolve field names based on their
 /// numbers during encoding.
 ///
-/// This type is public since it might be useful to clients who want to write
-/// advanced custom visitors.
-public enum ProtoNameResolvers {
+internal enum ProtoNameResolvers {
```

Comment:
We can always expose something later if there is reason, but once it is exposed, we're stuck until we want to break api, so I'd try to expose as little as possible.